scripts/scrape-roku-docs.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Added this because te linter was complaining that these dependencies should be `dependencies` instead of `devDependencies`

Reply:
yeah that's fine, this script is only run in dev mode anyway...